local_database/DataDumper/README.md

```diff
@@ -24,3 +24,19 @@ To connect to the database, your IP address will need to be added to the "allow"
 | `RESTORE_PORT`  | The port of the database to restore into.  | `5432`                         |
 | `RESTORE_DB_NAME` | The name of the database to restore into. | `source_collector_test_db`     |
 | `RESTORE_PASSWORD` | The password of the user to connect to the database. | `HanviliciousHamiltonHilltops` |
+
+
+```
+# .env
+
+DUMP_HOST="127.0.0.1"
+DUMP_USER="test_source_collector_user"
+DUMP_PASSWORD="HanviliciousHamiltonHilltops"
+DUMP_DB_NAME="source_collector_test_db"
+DUMP_PORT=5432
+RESTORE_HOST="127.0.0.1"
+RESTORE_USER="test_source_collector_user"
+RESTORE_PORT=5432
+RESTORE_DB_NAME="source_collector_test_db"
+RESTORE_PASSWORD="HanviliciousHamiltonHilltops"
+```
```

Comment:
I will give you a lot of leeway in your judgment on this, but:

1. We have an `ENV.md` file also in the DataDumper directory. Could this information be redundant with that? Should the two files be consolidated for convenience or kept separate for modularity?
2. I was torn on whether to maintain the `RESTORE_` environment variables, because technically these are automatically set in the docker-compose, so I didn't know if they needed to be specified. If you look at `docker-compose.yml`, you'll see the `DUMP_` envars are set, but the `RESTORE_` envars are not.